cpp/examples/abm_minimal.cpp

```diff
@@ -0,0 +1,153 @@
+/*
+* Copyright (C) 2020-2023 German Aerospace Center (DLR-SC)
+*
+* Authors: Khoa Nguyen
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#include "abm/abm.h"
+#include "abm/household.h"
+#include <cstdio>
+
+int main()
+{
+    // Set global infection parameters (similar to infection parameters in SECIR model) and initialize the world
+    mio::abm::GlobalInfectionParameters infection_params;
+
+    // Set same infection parameter for all age groups. For example, the incubation period is 4 days. 
+    infection_params.get<mio::abm::IncubationPeriod>() = 4.;
+    
+    // Create the world with infection parameters.
+    auto world = mio::abm::World(infection_params);
+
+    // There are 3 households for each hosehold group.
```

Comment:
```suggestion
    // There are 3 households for each household group.
```

Reply:
If this is copy and pasted from the abm, next time you can change that there too :-)